chrome/content/zotero/zoteroPane.js

```diff
@@ -5490,15 +5496,20 @@ var ZoteroPane = new function()
 			unNormalizedDirname = unNormalizedDirname.substring(0, lastSlash + 1);
 		}
 
-		let parts = OS.Path.split(syncedPath).components;
+		let parts = split(syncedPath);
 		for (let segmentsToDrop = 0; segmentsToDrop < parts.length; segmentsToDrop++) {
-			let correctedPath = OS.Path.join(basePath, ...parts.slice(segmentsToDrop));
+			let correctedPath = join(basePath, ...parts.slice(segmentsToDrop));
 
 			if (!(await OS.File.exists(correctedPath))) {
 				Zotero.debug('Does not exist: ' + correctedPath);
 				continue;
 			}
 			Zotero.debug('Exists! ' + correctedPath);
+			
+			if (Zotero.isWin) {
+				correctedPath = correctedPath.replace(/\//g, '\\');
+				Zotero.debug('Converted back to Windows path: ' + correctedPath);
+			}
```

Comment:
Why here and not above? Doesn't this mean we're now passing a path with forward slashes to `OS.File.exists()` on Windows? Does that work?

Reply:
Huh!